Formula/gtk4.rb

```diff
@@ -31,7 +33,10 @@ class Gtk4 < Formula
   depends_on "glib"
   depends_on "graphene"
   depends_on "hicolor-icon-theme"
+  depends_on "jpeg-turbo"
   depends_on "libepoxy"
+  depends_on "libpng"
+  depends_on "libtiff"
```

Comment:
Mandatory dependencies as of 4.5.0 https://gitlab.gnome.org/GNOME/gtk/-/blob/4.6.6/NEWS#L496. Used for some features.